Compilation error of tests

This is the error I get:
```
[ 28%] Built target OsqpEigen
Scanning dependencies of target MPCUpdateMatricesTest
[ 35%] Building CXX object tests/CMakeFiles/MPCUpdateMatricesTest.dir/MPCUpdateMatricesTest.cpp.o
In file included from /home/sdafarra/Software/googletest/build/install/include/gtest/gtest.h:62,
                 from /home/sdafarra/Software/robotology-superbuild/robotology/OsqpEigen/tests/MPCUpdateMatricesTest.cpp:9:
/home/sdafarra/Software/googletest/build/install/include/gtest/internal/gtest-internal.h:1402:53: error: static assertion failed: test_name must not be empty
 1402 |   static_assert(sizeof(GTEST_STRINGIFY_(test_name)) > 1,                      \
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~
/home/sdafarra/Software/googletest/build/install/include/gtest/gtest.h:2339:3: note: in expansion of macro ‘GTEST_TEST_’
 2339 |   GTEST_TEST_(test_suite_name, test_name, ::testing::Test, \
      |   ^~~~~~~~~~~
/home/sdafarra/Software/googletest/build/install/include/gtest/gtest.h:2345:42: note: in expansion of macro ‘GTEST_TEST’
 2345 | #define TEST(test_suite_name, test_name) GTEST_TEST(test_suite_name, test_name)
      |                                          ^~~~~~~~~~
/home/sdafarra/Software/robotology-superbuild/robotology/OsqpEigen/tests/MPCUpdateMatricesTest.cpp:186:1: note: in expansion of macro ‘TEST’
  186 | TEST(MPCTest,)
      | ^~~~
make[2]: *** [tests/CMakeFiles/MPCUpdateMatricesTest.dir/build.make:63: tests/CMakeFiles/MPCUpdateMatricesTest.dir/MPCUpdateMatricesTest.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:943: tests/CMakeFiles/MPCUpdateMatricesTest.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```
I am on Ubuntu 20.04.

--- a/tests/MPCTest.cpp
+++ b/tests/MPCTest.cpp
@@ -213,7 +213,7 @@ double getErrorNorm(const Eigen::Matrix<double, 12, 1> &x,
 }
 
 
-TEST(MPCTest,)
+TEST(MPCTest, nullptr)
 {
     // open the ofstream
     std::ofstream dataStream;

--- a/tests/MPCUpdateMatricesTest.cpp
+++ b/tests/MPCUpdateMatricesTest.cpp
@@ -183,7 +183,7 @@ void updateConstraintVectors(const Eigen::Matrix<double, 2, 1> &x0,
     upperBound.block(0,0,2,1) = -x0;
 }
 
-TEST(MPCTest,)
+TEST(MPCTest, nullptr)
 {
     // open the ofstream
     std::ofstream dataStream;

--- a/tests/QPTest.cpp
+++ b/tests/QPTest.cpp
@@ -10,7 +10,7 @@
 
 #include <OsqpEigen/OsqpEigen.h>
 
-TEST(QPProblem, )
+TEST(QPProblem, nullptr)
 {
     Eigen::Matrix2d H;
     H << 4, 1,

